### ISSUE TITLE
Grayish images during testing

### ISSUE THREAD

**Issue by AbdulMoqeet** [MIVRC/MSRN-PyTorch#7]
Hi ! 

When I trained a model with your [new MSRN] code on DIV2K using n-colors=3, it shows good validation accuracy but when I tested on Set5, the output images are grayish and PSNR is ~=10. 

Test command: 
 python main.py --degradation BI --data_test MyImage --scale 2 --model MSRN --pre_train ../model/model_best.pt --test_only --save_results --chop --save "MSRN" --testpath ../LR/LRBI --testset Set5

Train : I am using demo.sh for training, no other changes are made.

**Comment by CV-JunchengLi** [comment]
Sorry, I have never encountered this problem.
I also checked my code, our train and test codes are all based on RGB channels.
The channel of the dataset (Set5) you used is RGB?

**Comment by AbdulMoqeet** [comment]
I have resolved the problem.
Yes, I am using the RGB dataset. 
The thing is, I have trained my own model though during testing, I was using different model with different trained weights instead of that model which shown good results on DIV2K. ,
